src/main/java/net/minecraftforge/common/crafting/CraftingHelper.java

```diff
@@ -218,7 +218,13 @@ public static ItemStack getItemStack(JsonObject json, JsonContext context)
             // Lets hope this works? Needs test
             try
             {
-                NBTTagCompound nbt = JsonToNBT.getTagFromJson(GSON.toJson(json.get("nbt")));
+                JsonElement element = json.get("nbt");
+                NBTTagCompound nbt;
+                if(element.isJsonObject())
```

Comment:
oh I wasn't aware leaving {} off was accepted in the forge style. in that case, just ignore my original comment :P